packages/@okta/i18n/src/properties/login.properties

```
@@ -69,7 +69,7 @@ errors.E0000007 = Not found: {0}
 errors.E0000008 = The requested path was not found
 errors.E0000009 = Internal Server Error
 errors.E0000010 = Service is in read only mode
-errors.E0000011 = Invalid token provided
+errors.E0000011 = Your session has expired, Please refresh the page to sign in.
```

Can you update the tests ? 
https://github.com/okta/okta-signin-widget/blob/master/test/unit/helpers/xhr/ERROR_invalid_token.js#L6
https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/LoginRouter_spec.js#L849

---

Given the high visibility of this error message, let's have someone from doc 👍🏼  this new message.

```suggestion
errors.E0000011 = Your session has expired. Please refresh the page to sign in.
```

---

@grahamsmith-okta can you review the messaging around this ?

---

I would drop "has" and "please" - Your session expired. Refresh the page to sign in.
Also, in E000010 hyphenate read-only - Service is in read-only mode

---

addressed comments here on 5.4 base : https://github.com/okta/okta-signin-widget/pull/1733